packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/pools.html

```diff
@@ -0,0 +1,90 @@
+<div class="container-fluid px-0 mt-3">
+    <!-- error messages -->
+    <cui-message-container
+        data-messages="$ctrl.messages"
+    ></cui-message-container>
+
+    <h2 data-translate="pci_databases_pools_title"></h2>
+    <p data-translate="pci_databases_pools_description"></p>
+
+    <oui-datagrid
+        data-rows="$ctrl.pools"
+        data-empty-placeholder="{{ 'pci_databases_pools_retrieve_error' | translate }}"
```

Comment:
**suggestion:** one-time binding can be used

`data-empty-placeholder="{{ :: 'pci_databases_pools_retrieve_error' | translate }}"`